gprotorch/dataloader/reaction_loader.py

```diff
@@ -0,0 +1,130 @@
+import pandas as pd
+from gprotorch.data_featuriser import one_hot, rxnfp, drfp, bag_of_characters
+from gprotorch.dataloader import DataLoader
+
+
+class ReactionLoader(DataLoader):
```

Comment:
Looks great! I think it's definitely a good idea to have a separate dataloader for reactions otherwise if we made this part of the main DataLoader class things would quickly get messy!